skimage/exposure/exposure.py

```diff
@@ -144,15 +144,20 @@ def histogram(image, nbins=256, source_range='image', normalize=False):
     return hist, bin_centers
 
 
-def cumulative_distribution(image, nbins=256):
+def cumulative_distribution(image, nbins=256, source_range='image'):
```

Comment:
Can you make this a keyword-only argument?

```suggestion
def cumulative_distribution(image, nbins=256, *, source_range='image'):
```